### ISSUE TITLE
Change project name

### ISSUE THREAD

**Issue by taj-ny** [taj-ny/kwin-effects-forceblur#79]
Force blur isn't really an appropriate name anymore, since the effect does more than just that.

**Comment by D3SOX** [comment]
If you have decided on a new name tell me so I can rename the AUR package

**Comment by taj-ny** [comment]
I guess I'll just call it Better Blur (kwin-better-blur). Will the AUR package break again if I change the repository name?

**Comment by Eated-Universe** [comment]
Seems you reopened this.

Why not something simple and unique, such as "Blurry"?  If not, just keep it, 'better blur' is OK.
I you wanna mess with KDE naming, go for "blurK" or something lol

Some other I can drop of the top of my mind:

Blurio
Focal/Focaly
Blurix
KBlur
Blurz/Blurzy
BluRox/BluRocks
BlurFX
Blurify/Blurrify

**Comment by taj-ny** [comment]
> Seems you reopened this.

I was waiting for a response and forgot about it. I don't want to change the project name again, so I'll just keep the current one. Thanks for the suggestions though.

> Will the AUR package break again if I change the repository name?

@D3SOX

**Comment by D3SOX** [comment]
> > Seems you reopened this.
> 
> I was waiting for a response and forgot about it. I don't want to change the project name again, so I'll just keep the current one. Thanks for the suggestions though.
> 
> > Will the AUR package break again if I change the repository name?
> 
> @D3SOX 

No, if you only change the name github makes a redirect for it. But let me know if you did so anyways

**Comment by taj-ny** [comment]
I'll keep the repository name as it is, just in case changing it would affect the position in search engines.
